translationCoreApps/translationWords_Check_plugin
# Enhancement/ih 1489 tw names

#### This pull request addresses:

Addresses #1489 


#### How to test this pull request:

Make sure it opens.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/translationwords_check_plugin/83)
